.github/workflows/main.yml

```diff
@@ -15,9 +15,9 @@ jobs:
         with:
           node-version: '16'
       - run: npm ci
-      - run: ./node_modules/.bin/eslint src
+      - run: npm run lint
```

Comment:
Better to call the script than the linter directly - what if we changed the linter?